step.yml

```diff
@@ -304,19 +315,30 @@ inputs:
     is_required: true
 
 outputs:
+- BITRISE_TEST_BUNDLE_PATH:
+  opts:
+    title: Test Bundle directory
+    summary: Directory of the built targets' binaries and built associated tests.
+
+- BITRISE_TEST_BUNDLE_ZIP_PATH:
+  opts:
+    title: Zipped Test Bundle directory
+    summary: Zipped directory of the built targets' binaries and built associated tests.
+
+# TODO: is it used or is it a useful output?
```

Comment:
Do you think it is worth removing? I was hesitating on this, removing would mean a major version bump.

On the other side, `BITRISE_TEST_DIR_PATH` in itself (without the xctestrun file) is useless.

Reply:
Doing a major bump just for the sake of the `BITRISE_TEST_DIR_PATH` removal seems wasteful. But in this case you are doing a major revamp to make the step pipeline compatible. A major bump can be better sold with these changes. So just because of this I would be tempted to remove it.

Reply:
Removed in https://github.com/bitrise-steplib/steps-xcode-build-for-test/pull/39/commits/defe9983a98e85e1cd7dcd3cc6958eb9dff955c8 and https://github.com/bitrise-steplib/steps-xcode-build-for-test/pull/39/commits/64f640c9edaabaa2b29ddc7fd8e1fff394a8c576